pandas/tests/frame/test_constructors.py

```diff
@@ -2548,8 +2555,20 @@ def check_views(c_only: bool = False):
             check_views()
 
         # TODO: most of the rest of this test belongs in indexing tests
-        df.iloc[0, 0] = 0
-        df.iloc[0, 1] = 0
+        if (
+            is_float_dtype(df.dtypes.iloc[0])
```

Comment:
NBD but the suggested usage for these checks is now `lib.is_np_dtype(df.dtypes.iloc[0], "fciuomM")`

Reply:
nice, that simplifies things, thanks!